documentation/MeetingMinutes/2024/MOM-2024-01-31.md

```diff
@@ -0,0 +1,169 @@
+    - **Discussion on Issue #100 "Claims to be used in oAuth Access tokens".**
+    - **Discussion on Issue #108 "Question to documentation update for security schemes".**
+    - **Discussion on Issue #104 "Alignment of Client Credential authentication / authorization using API Gateway Pattern".**
+    - **Discussion on Issue #103 "Remove duplicate documentation after R0.1".**
+    - **Discussion on Issue #95 "Scope at "API level" feedback from country market implementation".**
+3. Action Points 
+4. AoB
+
+<br>
+
+## Recent updates & recap
+
+Telefónica presents the status of WG:
+
+- 2 issues closed since last call 17/01
+
+  - Issue #88 "Creating the first release for id and consent subproject"
+  - Issue #92 "Scope definition for Client Credentials"
+
+- 5 PR merged/closed since last call 17/01
+
+  - PR #74 "Consent related use cases (former PR#33)". Related with issue #21. Closed.
+  - PR #109 "Add rule for requesting scopes for client credentials". Fixes issue #92.
+  - PR #110 "IdentityAndConsentManagement meeting notes 2023-01-17"
+  - PR #111 "Fix outdated Meeting Minutes README"
+  - PR #112 "Update README and CHANGELOG for release v0.1.0". Fixes issue #88.
+
+- 12 ongoing issues:
+  
+  - Issue #78 "clarify the "credentials" notion in CIBA workflow authentication"
+  - Issue #82 "Redundancy with GSMA API General Federated Call Flows document"
+  - Issue #84 "Implement a Future-Proof Dynamic, Geolocation-Based User Consent Mechanism Compliant with Global Privacy Regulations" transferred from camaraproject/Governance.
+  - Issue #87 "Extending the technical scope from one technical scope to a parameters list of technical scopes."
+  - Issue #90 "OIDC guidance (what needs to be adhered to?)"
+  - Issue #95 "Scope at "API level" feedback from country market implementation"
+  - Issue #98 "Terms used in User Identity section in CAMARA-API-access-and-user-consent.md"
+  - Issue #99 "Clarify consent management functionality included in CAMARA-API-access-and-user-consent.md"
+  - Issue #100 "Claims to be used in oAuth Access tokens".
+  - Issue #103 "Remove duplicate documentation after R0.1"
+  - Issue #104 "Alignment of Client Credential authentication / authorization using API Gateway Pattern"
+  - Issue #108 "Question to documentation update for security schemes"
+
+- 1 ongoing PRs 
+  
+  - PR #105 "Update CAMARA-API-access-and-user-consent.md login_hint example". Fixes issue #104.
+
+- No new issues raised since last call 17/01
+
+  - N/A
+
+- 3 new PRs since last call 17/01
+
+  - PR #113 "Add CAMARA OIDC Profile". Fixes issues #78 and #90.
+  - PR #114 "Add missing response errors in CIBA flow from GSMA doc". Partially fixes issue #82.
+  - PR #115 "Delete documentation/CAMARA-AuthN-AuthZ-Concept.md". Fixes issue #103.
+
+<br>
+
+## New participant assignment to chair the working group.
+
+- It is announced that Axel Nennker (DT) will take over the chair from the next working group meeting. He replaces Jesús Peña (Telefónica).
```

Comment:
There is no "chair" role defined in Camara. This is the moderator role.

Reply:
It is clarified in line 99, as Axel did in the meeting itself. But I can add further clarifications to it that are not enough. This is just a naming issue in my opinion (my bad), nothing changes the expectations of the role responsibilities.